forge/Cargo.toml

```diff
@@ -18,16 +15,19 @@ serde = "1.0.130"
 regex = { version = "1.5.4", default-features = false }
 hex = "0.4.3"
 glob = "0.3.0"
-# TODO: Trim
+# TODO: Trim down
 tokio = { version = "1.10.1" }
 tracing = "0.1.26"
 tracing-subscriber = "0.2.20"
 proptest = "1.0.0"
 rayon = "1.5"
 rlp = "0.5.1"
 
-# load sputnik for parallel evm usage
-sputnik = { package = "evm", git = "https://github.com/rust-blockchain/evm" }
+bytes = "1.1.0"
+thiserror = "1.0.29"
+revm = { package = "revm", git = "https://github.com/onbjerg/revm", branch = "onbjerg/machine-exports" }
+hashbrown = "0.12"
```

Comment:
We need this because `revm` uses `HashMap` from hashbrown (to be `#[no_std]` compliant I guess?)